### PR TITLE
Add Elastic channels 

### PR DESCRIPTION
**Pull request by agarwhi**
**Issue #:**

**Description of changes:**
1. Add new feature : Elastic channels 
2. Fix "Missing region in Config" err
3. Fix issue of showing stale data on signIn 

**Testing**

1. How did you test these changes? Through Bug bash service team tested it
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?Yes, using "chat" demo 
3. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors?NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/chat/README.md [hunk 1]
```diff
@@ -113,14 +113,22 @@ Skip to [Creating a Channel](#creating-a-channel)
...
+  a. Select channel Type `Elastic` 
+  b. Enter values of `Maximum SubChannels`, `Target Memberships Per SubChannel` and `Scale-In Minimum Memberships`.
+
+5. Choose **Add**.
 
 **Sample code**
 
 ```js
-async function createChannel(appInstanceArn, name, mode, privacy, userId) {
+async function createChannel(appInstanceArn, name, mode, privacy, elasticChannelConfiguration, userId) {
```
Comment (devalevenkatesh) [inline apps/chat/README.md:131]: Any suggestion on what `elasticChannelConfiguration` looks like? would be good to have a example in code here.

File: apps/chat/src/components/ChannelModals/ChannelModals.css [hunk 2]
```diff
@@ -200,3 +212,19 @@ section .modal-header {
...
+  min-width: 35%;
+}
+
+.ch-form-field-input .value {
+  display: flex;
+  width: 100%;
+  margin: auto 0;
+  padding: 1rem;
+  flex-direction: column;
+}
```
Comment (devalevenkatesh) [inline apps/chat/src/components/ChannelModals/ChannelModals.css:230]: nit: Need newline at EOF

File: apps/chat/README.md [hunk 3]
```diff
@@ -240,6 +254,17 @@ The chat demo app also supports presence and typing indicators within a channel.
...
+2. Elastic channels make it easy to create secure, scalable, moderated chat experiences for large audiences which one can use with built in moderation features to help enforce brand, corporate, or community guidelines. 
+3. Elastic channels feature is available in the US East (N. Virginia) region and supports up to 1 Million members (see [Messaging Quotas](https://docs.aws.amazon.com/general/latest/gr//chime-sdk.html) for default values) 
+4. Existing channel flows feature can be used to automate moderation across all messages sent in the elastic channel.
+5. An elastic channel is a collection of subchannels with memberships distributed across subchannels.
+6. To create an elastic channel, provide channel name, privacy, and select channel Type `Elastic` and add desired value of `Maximum SubChannels`, `Target Memberships Per SubChannel` & `Scale-In Minimum Memberships`. Refer to [document](https://docs.aws.amazon.com/chime-sdk/latest/APIReference/API_messaging-chime_CreateChannel.html) for more details.
```
[the remark this answers is not in the document]
Comment (agarwhi) [reply apps/chat/README.md:263]: Updating it. Thanks

File: apps/chat/README.md [hunk 4]
```diff
@@ -113,14 +113,22 @@ Skip to [Creating a Channel](#creating-a-channel)
 ### Creating a channel
 
 1. Sign in to the client.
-1. In the **Channels** pane, choose the menu (•••).
-1. In the **Add channel** box, enter a channel name. Choose a desired type and mode.
-1. Choose **Add**.
+2. In the **Channels** pane, choose the menu (•••).
+3. In the **Add channel** box, enter a channel name. Choose a desired privacy and mode.
+4.
```
[the remark this answers is not in the document]
Comment (agarwhi) [reply apps/chat/README.md:118]: Yup..Correcting it

File: apps/chat/src/components/ChannelModals/ChannelModals.css (hunk 2 above)
Comment (agarwhi) [reply apps/chat/src/components/ChannelModals/ChannelModals.css:230]: Okay

File: apps/chat/README.md (hunk 1 above)
Comment (agarwhi) [reply apps/chat/README.md:131]: Addding

File: apps/chat/README.md [hunk 5]
```diff
@@ -113,14 +113,37 @@ Skip to [Creating a Channel](#creating-a-channel)
...
+  minimumMembershipPercentage
+) => {
+  return {
+    MaximumSubChannels: maximumSubChannels,
+    TargetMembershipsPerSubChannel: targetMembershipsPerSubChannel,
+    MinimumMembershipPercentage: minimumMembershipPercentage,
+  };
+};
+
+const elasticChannelConfiguration = getElasticChannelConfig(maximumSubChannels, targetMembershipsPerSubChannel, minimumMembershipPercentage);
```
Comment (devalevenkatesh) [inline apps/chat/README.md:144]: Sorry for not being clear you dont need the get function here simply below code should be fine:
```
const elasticChannelConfiguration = {
  MaximumSubChannels: <some number>,
  TargetMembershipsPerSubChannel: <value here>,
  MinimumMembershipPercentage: <value here>,
}
```

**Review by devalevenkatesh — DISMISSED** [review]

File: apps/chat/README.md (hunk 5 above)
Comment (agarwhi) [reply apps/chat/README.md]: updated.

**Review by devalevenkatesh — APPROVED** [review]
Please include the documentation link to elastic channels directly in README:
https://docs.aws.amazon.com/chime-sdk/latest/dg/elastic-channels.html

**Pull request merged**